components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java

```diff
@@ -157,6 +157,10 @@ private WSTrust(){
             public static final String CLEAN_UP_TIMEOUT_DEFAULT = "20160";
             public static final String CLEAN_UP_PERIOD = "CleanUpPeriod";
             public static final String CLEAN_UP_PERIOD_DEFAULT = "1140";
+            public static final String TIME_CONFIG = "TimeConfig";
+            private IDPProperties(){
```

Comment:
Yeah, that's right. I was referring to something similar to what's mentioned below.

private IDPProperties () {
throw new AssertionError();
}